static/elements/chromedash-legend.html

```diff
@@ -42,11 +42,11 @@
     <section>
       <label>Example search queries</label>
       <ul class="queries">
-        <li><span>"shipped_milestone&lt;30"</span>features that landed before 30</li>
-        <li><span>"shipped_android_milestone&gt;40"</span>features that landed on Anrdoid after 40 (milestone types: android, ios, webview, opera, opera_android)</li>
-        <li><span>"shipped_milestone&lt;=50"</span>features in desktop chrome 50</li>
+        <li><span>"browsers.chrome.desktop&lt;30"</span>features that landed before 30</li>
```

Comment:
This mechanism makes for very tight query parsing code, but I wonder whether it's the UX we want. Personally, I'm divided on the issue of "browsers.foo.bar":

Pros: Easy parsing, very precise, parsimonious if referring to other parts of the data model.

Cons: Expecting user to know about/use our data model, strange to qualify that the browser is Chrome on chromestatus.com, more verbose.

One option would be to support both; keep a map of "special" key shortcuts, and fallback on data-model-walking.

@ebidel and @jeffposnick, thoughts?